mlflow/transformers.py

```diff
@@ -1275,7 +1279,13 @@ def _get_default_pipeline_signature(pipeline, example=None) -> ModelSignature:
                         ColSpec("string", name="hypothesis_template"),
                     ]
                 ),
-                outputs=Schema([ColSpec("string")]),
+                outputs=Schema(
+                    [
+                        ColSpec("string", name="sequence"),
+                        ColSpec("string", name="labels"),
```

Comment:
```suggestion
                        ColSpec("string", name="label"),
```

Reply:
same as above (purely for consistency with the transformers API)